src/commands/pack.js

```diff
@@ -216,11 +216,12 @@ export default function (vorpal) {
       'pack',
       'Builds a grommet application for development and/or production'
     )
+    .option('--nobrowser', 'Skip browser opening on first compilation')
```

Comment:
Sure. There is only one little disadvantage: it's not possible to access this option with `options.skip-open`, only with `options['skip-open']`.